services/uploads/src/clamav.js

```diff
@@ -7,6 +7,26 @@ const utils = require("./utils");
 
```

Comment:
(General comment)
I'm taking all the *.js changes on faith; I imagine they came from mcr and are drag and drop.  Looks fine to me.